scm-webapp/src/main/java/sonia/scm/api/v2/resources/BrowserResultToFileObjectDtoMapper.java

```diff
@@ -58,6 +60,9 @@ FileObjectDto map(BrowserResult browserResult, NamespaceAndName namespaceAndName
   @Children
   protected abstract FileObjectDto fileObjectToDto(FileObject fileObject, @Context NamespaceAndName namespaceAndName, @Context BrowserResult browserResult, @Context Integer offset);
 
+  @Children
+  abstract List<FileObjectDto> map(Collection<FileObject> value);
+
```

Comment:
It looks like this change is not sufficient for the mapping of file trees. All integration tests fail for reading files.